<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryManager</class>
 <widget class="QWidget" name="LibraryManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">border:none;
background:rgb(236,236,236)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget1" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(235, 235, 235, 255), stop:1 rgba(170, 170, 170, 255));</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>2</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../Resouce.qrc">:/images/库管理小图标.png</pixmap>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>2</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
font: 11pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string>库管理</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget2" native="true">
        <property name="minimumSize">
         <size>
          <width>1080</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1080</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>库编号</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>30</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>库名称</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="m_editLibNumber">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>30</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;
background-color:rgb(255,255,255)</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="m_editLibName">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>30</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;
background-color:rgb(255,255,255)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="m_btnSearch">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>30</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;</string>
         </property>
         <property name="text">
          <string>搜索</string>
         </property>
        </widget>
        <widget class="QPushButton" name="m_btnAddLib">
         <property name="geometry">
          <rect>
           <x>744</x>
           <y>22</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;</string>
         </property>
         <property name="text">
          <string>+ 添加库</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="m_table"/>
      </item>
      <item>
       <widget class="QWidget" name="widget3" native="true">
        <property name="minimumSize">
         <size>
          <width>1080</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1080</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QPushButton" name="m_btnSelectAll">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>52</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;
width:50;
height:20;
background-color: rgb(222, 222, 222);
QPushButton{
hover:background-color: rgb(41, 147, 207);
pressed:background-color: rgb(41, 147, 207);
}</string>
           </property>
           <property name="text">
            <string>全选</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnReSelectAll">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>52</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;
width:50;
height:20;
background-color: rgb(222, 222, 222);
QPushButton{
hover:background-color: rgb(41, 147, 207);
pressed:background-color: rgb(41, 147, 207);
}</string>
           </property>
           <property name="text">
            <string>反选</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnDelete">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>52</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;
width:50;
height:20;
background-color: rgb(222, 222, 222);
QPushButton{
hover:background-color: rgb(41, 147, 207);
pressed:background-color: rgb(41, 147, 207);
}</string>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>625</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnPageFirst">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;
width:20;
height:20;
background-color: rgb(222, 222, 222);
QPushButton{
hover:background-color: rgb(41, 147, 207);
pressed:background-color: rgb(41, 147, 207);
}
</string>
           </property>
           <property name="text">
            <string>&lt;&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnPagePre">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;
width:20;
height:20;
background-color: rgb(222, 222, 222);
QPushButton{
hover:background-color: rgb(41, 147, 207);
pressed:background-color: rgb(41, 147, 207);
}</string>
           </property>
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnPageOne">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;
width:20;
height:20;
background-color: rgb(222, 222, 222);
QPushButton{
hover:background-color: rgb(41, 147, 207);
pressed:background-color: rgb(41, 147, 207);
}</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnPageTwo">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;
width:20;
height:20;
background-color: rgb(222, 222, 222);
QPushButton{
hover:background-color: rgb(41, 147, 207);
pressed:background-color: rgb(41, 147, 207);
}</string>
           </property>
           <property name="text">
            <string>2</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnPageThree">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;
width:20;
height:20;
background-color: rgb(222, 222, 222);
QPushButton{
hover:background-color: rgb(41, 147, 207);
pressed:background-color: rgb(41, 147, 207);
}</string>
           </property>
           <property name="text">
            <string>3</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_labelPage">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;黑体&quot;;
border:none</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnPageFive">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;
width:20;
height:20;
background-color: rgb(222, 222, 222);
QPushButton{
hover:background-color: rgb(41, 147, 207);
pressed:background-color: rgb(41, 147, 207);
}</string>
           </property>
           <property name="text">
            <string>5</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnPageNext">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;
width:20;
height:20;
background-color: rgb(222, 222, 222);
QPushButton{
hover:background-color: rgb(41, 147, 207);
pressed:background-color: rgb(41, 147, 207);
}</string>
           </property>
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnPageLast">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(180,180,180);
border-radius:2px;
width:20;
height:20;
background-color: rgb(222, 222, 222);
QPushButton{
hover:background-color: rgb(41, 147, 207);
pressed:background-color: rgb(41, 147, 207);
}</string>
           </property>
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resouce.qrc"/>
 </resources>
 <connections/>
</ui>
